### ISSUE TITLE
hp::FECollection::find_common_subspace still in docs

### ISSUE THREAD

**Issue by drwells** [dealii/dealii#8255]
@marcfehling I was reviewing the changelog when I noticed that this former function name is still in the documentation. Do you know what replaced it?

**Comment by drwells** [comment]
I think its `find_common_fes`.

**Comment by marcfehling** [comment]
Yes, we renamed it `find_common_fes`. I thought I updated the documentation accordingly :-/

If I provide a PR on this issue, I will have to merge it with the `dealii-9.1` branch, correct?

**Comment by drwells** [comment]
Yes; it would be great if we could cherry-pick it onto a second PR that is against `dealii-9.1`. Do you know how to do that? I can help if not.

**Comment by drwells** [comment]
I am working my way through `9.0.1-vs-9.1.0.h` now so you can skip that one.

**Comment by marcfehling** [comment]
I adjusted the documentation in this commit https://github.com/marcfehling/dealii/commit/4162a8339e3ad152943648c9ca42480552ea9db1

I don't know how to cherry-pick it onto a different PR. Also, I am just have the `Contributor` status and I don't know if I'm able to do that. Could you assist me on that?

**Comment by drwells** [comment]
Sure. It doesn't involve your GitHub status, you just have to set up the PR another way.

Consider #8257: I want it to also show up on 9.1. I created a new branch off of dealii-9.1:
```
git checkout master
git fetch upstream dealii-9.1
git checkout -b add-class-doc-stubs-9.1
git cherry-pick 53bae7ac4f77
git push origin add-class-doc-stubs-9.1
```
I now have a branch based on the 9.1 branch which contains the new commit. I can then open a PR in the usual way *but* I have to set the base fork branch to `dealii-9.1`.

Does this make sense?

**Comment by marcfehling** [comment]
Okay, that is exactly what I planned to do :)

**Comment by drwells** [comment]
Fixed by #8260 and #8261.
